Package.swift

```diff
@@ -5,8 +5,8 @@ import PackageDescription
 let package = Package(
         name: "ConcordiumSwiftSDK",
         platforms: [
-            .macOS(.v10_15),
             .iOS(.v15),
+            .macOS(.v10_15),
```

Comment:
Agree, a comment would only be (potentially) useful in the case where we have a have a single dependency/reason that forces us to set a specific minimum version, so I am completely fine with leaving it out, so that the comment doesn't start lying the second we have additional reasons to bump the version further.